src/rgw/rgw_admin.cc

```diff
@@ -5213,12 +5213,6 @@ int main(int argc, const char **argv)
   }
 
   if (opt_cmd == OPT_OLH_GET) {
-    RGWBucketInfo bucket_info;
-    int ret = init_bucket(tenant, bucket_name, bucket_id, bucket_info, bucket);
-    if (ret < 0) {
-      cerr << "ERROR: could not init bucket: " << cpp_strerror(-ret) << std::endl;
-      return -ret;
-    }
```

Comment:
After recheck I've found the global ` RGWBucketInfo bucket_info` didn't has been initialized and used.

Each if (cmd == XXX ) {} block that initialize the local var
    `RGWBucketInfo bucket_info` explicitly by invoking `init_bucket()`.  Maybe we need to stick to this pattern for consistency.

repushed. Plz review again.